<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyInfoWidget</class>
 <widget class="QWidget" name="ModifyInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_modifyInfo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>150</y>
     <width>10</width>
     <height>600</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lb_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_sex">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sex">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mojor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>专业</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_major">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_major_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>640</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mojor_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifyInfo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>740</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_modifyPassword">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>90</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_password">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>220</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_password">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>160</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_confirmPassword">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_confirmPassword">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>340</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifyPassword">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>740</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
